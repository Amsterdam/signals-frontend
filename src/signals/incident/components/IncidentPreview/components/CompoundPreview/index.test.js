import React from 'react';
import { shallow } from 'enzyme/build';
import { createCompoundPreview } from './index';

describe('the compound preview component', () => {
  it('should wrap two different components in a div', () => {
    const Compound = createCompoundPreview([
      () => <span>hello world</span>,
      foo => (
        <span>
bar:
          {foo}
        </span>
      ),
    ]);

    const wrapper = shallow(<Compound abc="xyz" />);

    expect(wrapper).toMatchSnapshot();
  });
});
