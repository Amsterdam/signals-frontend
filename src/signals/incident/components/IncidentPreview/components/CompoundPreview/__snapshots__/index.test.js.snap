// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the compound preview component should wrap two different components in a div 1`] = `
<div>
  <Component
    abc="xyz"
    key="0"
  />
  <Component
    abc="xyz"
    key="1"
  />
</div>
`;
